src/cf-forms/src/molecules/field.less

```diff
@@ -22,6 +22,7 @@
                 width: unit( 18px / @base-font-size-px, em );
                 margin-right: 10px;
 
+                background-color: @input-bg;
                 content: '';
                 vertical-align: top;
```

Comment:
Found a little rendering error when checked and focused in Firefox:

![screen shot 2016-09-21 at 13 05 23](https://cloud.githubusercontent.com/assets/1044670/18721633/516eb690-7fff-11e6-86d8-dd6828ef3f9e.png)

A possible fix, if a little magic-y, is to add `line-height: 1.25` here. Tested successfully in Firefox and Chrome so far. Has the added benefit of slightly better vertical centering of the checkmark.